test/test_emissions.py

```diff
@@ -7,51 +7,88 @@
 @pytest.fixture
 def emissions():
     """Need to provide this import as a fixture to avoid complaints from the linter."""
-    sys.path.append('../')
+    sys.path.append("../")
     import src.emissions as emissions
+
     return emissions
 
 
 @pytest.fixture
 def df_co2_ch4_no2():
-    df = pd.DataFrame({
-        'co2_mass_lb': [1.0, 2.0],
-        'ch4_mass_lb': [3.0, 6.0],
-        'n2o_mass_lb': [0.5, 0.2],
-        'co2_mass_lb_adjusted': [1.0, 2.0],
-        'ch4_mass_lb_adjusted': [3.0, 6.0],
-        'n2o_mass_lb_adjusted': [0.5, 0.2],
-        'co2_mass_lb_for_electricity': [1.0, 2.0],
-        'ch4_mass_lb_for_electricity': [3.0, 6.0],
-        'n2o_mass_lb_for_electricity': [0.5, 0.2]
-    })
+    df = pd.DataFrame(
+        {
+            "co2_mass_lb": [1.0, 2.0],
+            "ch4_mass_lb": [3.0, 6.0],
+            "n2o_mass_lb": [0.5, 0.2],
+            "co2_mass_lb_adjusted": [1.0, 2.0],
+            "ch4_mass_lb_adjusted": [3.0, 6.0],
+            "n2o_mass_lb_adjusted": [0.5, 0.2],
+            "co2_mass_lb_for_electricity": [1.0, 2.0],
+            "ch4_mass_lb_for_electricity": [3.0, 6.0],
+            "n2o_mass_lb_for_electricity": [0.5, 0.2],
+        }
+    )
     return df
 
 
+# NOTE: Why are we testing these function specifically?
```

Comment:
Do we need to answer this question?

Reply:
Good question. I think @miloknowles had implemented all of our original tests, and maybe this is just where he got started or these were the functions he was using at the time. Perhaps in the future we can work on improving our test coverage, but we also have a lot of validation checks built in.

Reply:
I think I had added these tests when we originally added those CO2-eq helpers - feel free to delete if you don't think we need them or if they're adding overhead to maintain